bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/overheads/Overheads.scala

```diff
@@ -41,9 +37,15 @@ object Overheads extends (BlueprintBase => Option[Overheads]) {
     }
   }
 
-  private case class IGRINS2Overheads(telluricTime: TimeAmount, nTelluric: Int) extends Overheads {
+  private case class IGRINS2Overheads(telluricTime: TimeAmount, telluricStars: Igrins2TelluricStars) extends Overheads {
     override def calculate(progTime: TimeAmount): ObservationTimes = {
       val progTimeHrs: Double = progTime.toHours.value
+      val nTelluric = telluricStars match {
+        case Igrins2TelluricStars.Default  => (progTimeHrs / 1.5).toInt
```

Comment:
What happens when `progTimeHrs` is >= 4.5 hours?  That is, is it allowed to be > 2?  Seems like it could be rounded as well?

Reply:
There is no upper bound, if you ask for 6h you'd get 4 stars.
It seems most observations are short though